app/templates/components/gtm.html

```diff
@@ -0,0 +1,26 @@
+{#
+This component enables Google Tag Manager.
+It should be included in the base template if the user has enabled tracking cookies.
+#}
+
+{% if "cookies_policy" in request.cookies %}
+{% if request.cookies.cookies_policy | dict | get_item("analytics") == "yes" %}
+
+{% set gtm_measurement_id = 'G-S8M7DQZ6N0' %}
```

Comment:
On GTM we have GTM-PJPFZVSS as the Tagman id for Access

Reply:
Thanks Jav, great spot. This is resolved with the latest commit.